packages/pn-pa-webapp/src/components/NewNotification/Recipient.tsx

```diff
@@ -505,152 +379,129 @@ const Recipient: React.FC<Props> = ({
                         errors={errors}
                         setFieldValue={setFieldValue}
                         handleBlur={handleBlur}
-                        width={12}
+                        sx={{ marginTop: 2 }}
                       />
-                      {paymentMode !== PaymentModel.NOTHING && (
-                        <>
+                    </Grid>
+                    <Grid item lg={4} sx={{ display: { xs: 'none', lg: 'block' } }}></Grid>
```

Comment:
why this empty grid item?

Reply:
To respect input width across various screen sizes. But I removed it and added a new grid